### ISSUE TITLE
Not working with MIUI 12

### ISSUE THREAD

**Issue by jamesjguthrie** [codejamninja/react-native-exposure-notification#2]
Protect Scotland is using this package and that app reports in MIUI 12 that Play Services must be updated to use Protect Scotland (even with the latest framework installed). I believe this is caused by the useExposure function returning a false/negative result.

MIUI 12 with Play Services installed does support the Exposure Notifications API. I have confirmed this with Italy's Immuni app.

**Comment by clayrisser** [comment]
@Jabo-SCO do you have access to the exposure notification api?

**Comment by jamesjguthrie** [comment]
With Immuni I do. Immuni is confirmed installed in the Exposure Notifications section of the Play Services settings. Immuni does not use your package, it has another approach. Immuni is open sourced on GitHub if you want to check it.

**Comment by clayrisser** [comment]
@Jabo-SCO can you send me the link to it. Is it using react native?

I would love to get this project integrated officially with the exposure notification api so others can easily integrate it with react native apps. I built this based on a publicly released specification, so it is more psuedo code that actually functional. Because I do not have access to the Exposure Notification api I cannot proceed further.

If you have the ability to create a pull request to fill in the missing peicies, or know someone who can, that would be extremely helpful. Another option would be if someone is willing to share access to the api with me.

**Comment by jamesjguthrie** [comment]
@codejamninja Protect Scotland, using react native and react-native-exposure-notification, DOESN'T work in MIUI 12, source code is here https://github.com/NES-Digital-Service/protect-scotland

Immuni is not using react native, DOES work in MIUI 12, its source code is here https://github.com/immuni-app/immuni-app-android

**Comment by jamesjguthrie** [comment]
I could submit a pull request on Protect Scotland, or you could, if you can identify where the issue lies - either within this framework or the app itself (I doubt the latter).

**Comment by clayrisser** [comment]
Yeah it's really hard for me to identify when I can't see the API. It's like flying blind.

**Comment by jamesjguthrie** [comment]
If you clone Protect Scotland you'd have access to the Exposure Notifications API right?

**Comment by clayrisser** [comment]
You have to have developer access, which is only given out to specific organizations. One of those organizations would have to create an account for me or give me developer access.
